export default {
  translation: {
    processError: {
      network: 'Network Problems. Try again.',
      parsing: 'This feed does not validate.',
      unexpected: 'An unexpected error occurred.',
    },
    validationError: {
      notOneOf: 'The resource already exists.',
      url: 'The resource must be a valid URL.',
    },
  },
};
